### PR TITLE
Traffic Settings: add Jetpack Search upgrade message

### PR DESCRIPTION
**Pull request by gibrown**
This adds an upgrade message in traffic settings where the Jetpack Search settings are displayed once a plan has been purchased.

The messaging is based on https://github.com/Automattic/jetpack/issues/9747

I have also reordered the traffic settings page a bit to adjust which items are on top. Some thoughts behind this:
- Put the things the user is more likely to toggle higher up - things that are not on by default for instance
- Put stuff like disabling stats at the bottom since very few folks disable stats

One question I have here is that I am seeing "Warning: Failed prop type: Invalid prop `description` supplied to `Banner`." in my console (Chrome). I suspect this has to do with the gnarly html description I have used. Some way around this?

### CODE REVIEW

**Comment by ebinnion** [comment]
Top of traffic tab for Jetpack site:

screen shot 2018-09-07 at 2 25 02 pm

Top of traffic tab for WP.com site:

screen shot 2018-09-07 at 2 24 26 pm

**Comment by ebinnion** [comment]
I pushed a commit to address the prop warning. Besides that, the code looks OK to me. We should probably get a design review for the content in the banner since it seems like more than usual.

File: client/my-sites/site-settings/search.jsx [hunk 1]
```diff
@@ -152,11 +153,39 @@ class Search extends Component {
...
+						plan={ siteIsJetpack ? PLAN_JETPACK_BUSINESS : PLAN_BUSINESS }
+						title={ upgradeTitle }
+					/>
+				</div>
+			);
 		}
 
 		return (
-			<div>
+			<div style={ { marginBottom: '16px' } }>
```
Comment (eliorivero) [inline client/my-sites/site-settings/search.jsx]: This should be moved to a stylesheet.

**Comment by michelleweber** [comment]
I'd tweak the headline: Add faster, more advanced searching to your site with Jetpack Professional

In the body copy: 
- I'd delete the first sentence entirely ("WP build in search is fine for small sites..."). Noneed to slag WP in our own UI.
- "High-relevant" needs a hyphen here, but I'd reconsider it As written, it sounds like the search engine is highly relevant, rather than that it delivers highly relevant results. Since you make the point about relevancy in the next sentence, I'd delete the "highly-relevant' bit.
- Delete "that is"

**Comment by gibrown** [comment]
@ebinnion or @eliorivero can I get one last look. Only changes I made were to move the style into into the css sheet and some updates to the wording.

@jeffgolenski fyi, @michelleweber helped a bit with some feedback on the wording and I'm going with that. The latest looks like:

screen shot 2018-09-13 at 2 53 02 pm

I'll use the same wording in the Jetpack PR for an upgrade message.

**Pull request merged**